packages/hash/subgraph/src/types/identifier.ts

```diff
@@ -35,27 +35,41 @@ export const extractEntityUuidFromEntityId = (entityId: EntityId): string => {
 /** @todo - consider Type Branding this */
 export type Timestamp = string;
 
-// ISO-formatted datetime string
-export type EntityVersion = Timestamp;
+export type VersionTimespan = {
+  start: Timestamp;
+  end?: Timestamp;
+};
+
+export type EntityVersion = {
+  decisionTime: VersionTimespan;
+  transactionTime: VersionTimespan;
```

Comment:
The more I see these the less I'm convinced by our naming, but I haven't managed to wrap my head around what we _should_ call it yet. 

The decision time (in my mind) is just the start _timestamp_, it's the _time_ at which you _decided_ it. 

There's then an associated time_span_ from that decision to the next decision.

Reply:
We could call it (applies to all locations) `decisionTimeRange`, but I don't really like that either.

Reply:
`decisionTimeValidRange`?

Reply:
`decisionTimeActiveRange`?